### PR TITLE
feat: ToastNotificationActivatedEventArgs structs for all

### PR DESCRIPTION
**Pull request by pkar70**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?
No ToastNotificationActivatedEventArgs implementation.

## What is the new behavior?
ToastNotificationActivatedEventArgs implemented.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information
Two more structs/classes from my #2585 - these structs/classes are defined for all platforms.

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UWP/ApplicationModel/Activation/ToastNotificationActivatedEventArgs.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+namespace Windows.ApplicationModel.Activation
+{
+	public partial class ToastNotificationActivatedEventArgs : global::Windows.ApplicationModel.Activation.IToastNotificationActivatedEventArgs, global::Windows.ApplicationModel.Activation.IActivatedEventArgs, global::Windows.ApplicationModel.Activation.IActivatedEventArgsWithUser, global::Windows.ApplicationModel.Activation.IApplicationViewActivatedEventArgs
```
Comment (MartinZikmund) [inline src/Uno.UWP/ApplicationModel/Activation/ToastNotificationActivatedEventArgs.cs:3]: Remove the namespaces and add a using instead 👍

File: src/Uno.UWP/ApplicationModel/Activation/IToastNotificationActivatedEventArgs.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+namespace Windows.ApplicationModel.Activation
+{
+
+	public partial interface IToastNotificationActivatedEventArgs : global::Windows.ApplicationModel.Activation.IActivatedEventArgs
```
Comment (MartinZikmund) [inline src/Uno.UWP/ApplicationModel/Activation/IToastNotificationActivatedEventArgs.cs:4]: Same here 😉

**Review by MartinZikmund — CHANGES_REQUESTED** [review]

File: src/Uno.UWP/ApplicationModel/Activation/ToastNotificationActivatedEventArgs.cs (hunk 1 above)
Comment (pkar70) [reply src/Uno.UWP/ApplicationModel/Activation/ToastNotificationActivatedEventArgs.cs]: Done

File: src/Uno.UWP/ApplicationModel/Activation/IToastNotificationActivatedEventArgs.cs (hunk 2 above)
Comment (pkar70) [reply src/Uno.UWP/ApplicationModel/Activation/IToastNotificationActivatedEventArgs.cs]: Done

File: src/Uno.UWP/ApplicationModel/Activation/ToastNotificationActivatedEventArgs.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+namespace Windows.ApplicationModel.Activation
+{
+	public partial class ToastNotificationActivatedEventArgs : IToastNotificationActivatedEventArgs, IActivatedEventArgs, IActivatedEventArgsWithUser, IApplicationViewActivatedEventArgs
+	{
+		public string Argument { get; internal set; }
```
Comment (jeromelaban) [inline src/Uno.UWP/ApplicationModel/Activation/ToastNotificationActivatedEventArgs.cs:5]: It's generally best to use a constructor for this:

```suggestion
		internal ToastNotificationActivatedEventArgs (string argument) => Argument = arguent;
		public string Argument { get; }

```

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UWP/ApplicationModel/Activation/ToastNotificationActivatedEventArgs.cs (hunk 3 above)
Comment (pkar70) [reply src/Uno.UWP/ApplicationModel/Activation/ToastNotificationActivatedEventArgs.cs:5]: As you wish, although my VStudio suggests "IDE0021 Use block body for constructors" :)

**Comment by pkar70** [comment]
Please restart build, as failure is (I think) unrelated to PR changes.

**Comment by pkar70** [comment]
azp /run

**Comment by pkar70** [comment]
(just try to restart build :) )

**Comment by pkar70** [comment]
Can this build be restarted?

**Review by davidjohnoliver — APPROVED** [review]

**Comment by pkar70** [comment]
It fails on WASM tests, screenshot compare. But how can it be related to this PR, implementing only struct, no real code?

**Comment by davidjohnoliver** [comment]
> It fails on WASM tests, screenshot compare. But how can it be related to this PR, implementing only struct, no real code?

It's a problem with the test runner itself, it's unrelated to the PR.

**Pull request merged**